### PR TITLE
Created Rslurm example code

### PR DESCRIPTION
**Pull request by jennylsmith**
Added a demo on using Rslurm for bioinformatics application.

### CODE REVIEW

File: _compdemos/rslurm_example.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+- You will need access to the Rhinos for submitting the SBATCH job (otherwise, you can just skip that step)
+
+
+More information on `Rslurm` is on its [vignette](https://cran.r-project.org/web/packages/rslurm/vignettes/rslurm.html) and `tximport` also has a nice [vignette](https://bioconductor.org/packages/devel/bioc/vignettes/tximport/inst/doc/tximport.html).
+
+
+# Instructions on how to get set-up
+
+1. open a terminal
+2. ssh user@rhino
```
Comment (TapscottLab) [inline _compdemos/rslurm_example.md]: If X11 forwarding is needed, should we do: ssh -X -Y user@rhino?

File: _compdemos/rslurm_example.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+Be sure to set your working directory to a place where you have a little space, and read-write access. I usually use scratch.
+
+```
+# knitr::opts_knit$set(root.dir = '/fh/scratch/delete10/PI_NAME/')
+```
+
+Install tximport if necessary and load it into your R session.
+
+```
+# BiocManager::install("tximport")
```
Comment (TapscottLab) [inline _compdemos/rslurm_example.md]: I would put this before installing tximport:
```
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
```

File: _compdemos/rslurm_example.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+Rslurm will create a directory containg all the accessory data files, libraries to load, and the batch script for you. I have `submit=TRUE` assuming you are running this on a Rhino/GIZMO node.
+
+## Submit Job to Scheduler
+
+```
+txi.geneLevel.job <- slurm_call(f=tximport, #function to be used
+                     jobname = "tximport_Gene", #name of the  output directory
+
+                    #Define parameters for the function here, based on the exact argument names, etc in the ?tximport documentation.
+                     params =  list(files = files,
```
Comment (TapscottLab) [inline _compdemos/rslurm_example.md]: Question: Is the `params` argument the container for the input argument for `tximport`?

File: _compdemos/rslurm_example.md (hunk 1 above)
Comment (atombaby) [reply _compdemos/rslurm_example.md]: No, as a rule one or the other is sufficient.  @vortexing has opened an issue for us (scicomp) to figure out which to recommend.

File: _compdemos/rslurm_example.md (hunk 3 above)
Comment (vortexing) [reply _compdemos/rslurm_example.md]: Yup!  Discussed the other day.  Going to close this.

**Comment by vortexing** [comment]
@jennylsmith do you want to add that little example blurb up there into this or do you want to merge this pull request?

**Comment by atombaby** [comment]
I'd like to suggest that we punt on the whole access issue (i.e. starting and running rstudio) as this is covered in greater detail elsewhere:

https://sciwiki.fredhutch.org/compdemos/howtoRhino/#how-to-run-gui-x-windowsx11-apps-on-rhino-and-gizmo-nodes

https://sciwiki.fredhutch.org/scicomputing/software_R/

https://sciwiki.fredhutch.org/scicomputing/access_methods/

**Pull request merged**